zarr/store/directory.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+import os
+import shutil
+import json
+
+
+import numpy as np
+
+
+from zarr.store.base import ArrayStore
+from zarr.util import normalize_shape, normalize_chunks, normalize_cparams, \
+    normalize_resize_args
+from zarr.compat import PY2
+from zarr.mappings import frozendict, Directory
+
+
+METAPATH = '__zmeta__'
+DATAPATH = '__zdata__'
+ATTRPATH = '__zattr__'
+
+
+class DirectoryStore(ArrayStore):
+
```

Comment:
Is it possible to remove the need to create a special subclass for each kind of `MutableMapping`?